src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -14,7 +14,11 @@ Welcome to the LiveChat API documentation!
 
 Our API provides a set of flexible tools which you can use to create new outstanding projects. We smile a bit more each time we see skilled developers unleash their creativity!
 
-Please note that this documentation refers to the **API version 2.0**. This is the **legacy** version. We encourage you to migrate to the latest stable version. [Read more about versioning...](/management/#api-versioning)
+<Warning>
+
+Please note that this documentation refers to the **API version 2.0**. This is the **deprecated** version, and you'll maintain access only to methods used in the last 30 days. We encourage you to migrate to the latest stable version. [Read more about versioning...](/management/#api-versioning)
```

Comment:
When someone enters the documentation later than on the day of closing v2, the "last 30 days" will no apply. You know what I mean? I wouldn't mention it I guess

Reply:
Okay, I get it. You're right. Haven't thought about it this way. I'll remove it